src/features/people/PeopleList/index.js

```diff
@@ -1,45 +1,47 @@
-import { MainWrapper, Header, TilesContainer, TilePerson, ImageWrapper, Image, Title } from "./styled";
+import { MainWrapper, Header, TilesContainer, TilePerson, ImageWrapper, Image, Title, StyledLink } from "./styled";
 import { useEffect } from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { fetchPeople, selectError, selectLoading, selectPeople} from "../peopleSlice";
+import { fetchPeople, selectError, selectLoading, selectPeople } from "../peopleSlice";
 import Error from "../../../common/Error";
 import Loading from "../../../common/Loading";
+import { toPersonDetails } from "../../../routes";
 
 const PeopleList = ({ insideDetails }) => {
-    const people = useSelector(selectPeople);
-    const loading = useSelector(selectLoading);
-    const error = useSelector(selectError);
-    const dispatch = useDispatch();
+  const people = useSelector(selectPeople);
+  const loading = useSelector(selectLoading);
+  const error = useSelector(selectError);
+  const dispatch = useDispatch();
 
-    useEffect(() => {
-        dispatch(fetchPeople());
-    }, [dispatch]);
+  useEffect(() => {
+    dispatch(fetchPeople());
+  }, [dispatch]);
 
-    if (error) {
-        return (
-            <Error />
-        );
-    }
-    return (
-        <MainWrapper insideDetails={insideDetails}>
-            {loading ? (<Loading />)
-                :
-                (<>
-                    <Header>Popular people</Header>
-                    <TilesContainer>
-                        {people.map(person => 
-                            <TilePerson key={person.id}>
-                                <ImageWrapper>
-                                    <Image src={`https://image.tmdb.org/t/p/w185${person.profile_path}`} alt="" />
-                                </ImageWrapper>
-                                <Title>{person.name}</Title>
-                            </TilePerson>
-                        )}
-                    </TilesContainer>
-                </>
-                )}
-        </MainWrapper>
-    )
+  if (error) {
+    return <Error />;
+  }
+  return (
+    <MainWrapper insideDetails={insideDetails}>
+      {loading ? (
+        <Loading />
+      ) : (
+        <>
+          <Header>Popular people</Header>
+          <TilesContainer>
+            {people.map((person) => (
+              <StyledLink key={person.id} to={toPersonDetails({ id: person.id })}>
+                <TilePerson key={person.id}>
+                  <ImageWrapper>
+                    <Image src={`https://image.tmdb.org/t/p/w185${person.profile_path}`} alt="" />
```

Comment:
Here maybe we can also added the same ternary operator as MovieList?

Reply:
Done. I will also add similar ternary for PersonDetail when I will add logic to the component.